pkg/wsman/amt/boot/decoder.go

```diff
@@ -55,3 +60,17 @@ func (i IDERBootDevice) String() string {
 
 	return ValueNotFound
 }
+
+func DecodeWSMANError(subCode string) error {
```

Comment:
Is this just for this package? do we want to centralize an error handling function? or are they specific enough to warrant a function per package.

Reply:
centralized!